extensions/wikia/WikiaNewtalk/wikia_newtalk.php

```diff
@@ -130,6 +140,7 @@ function wfGetWikiaNewtalk( User $user, &$talks ) {
 				__METHOD__
 			);
 			while( $row = $dbr->fetchObject( $sth ) ) {
+				// TODO: use GlobalTitle
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)